audit.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
should it be merged?

Reply:
This file contains all the false positives that ZAP is reporting